addons/point_of_sale/models/pos_order.py

```diff
@@ -304,20 +304,16 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     is_invoiced = fields.Boolean('Is Invoiced', compute='_compute_is_invoiced')
     is_tipped = fields.Boolean('Is this already tipped?', readonly=True)
     tip_amount = fields.Float(string='Tip Amount', digits=0, readonly=True)
-    refund_orders_count = fields.Integer('Number of Refund Orders', compute='_compute_refund_related_fields')
     is_refunded = fields.Boolean(compute='_compute_refund_related_fields')
-    refunded_order_ids = fields.Many2many('pos.order', compute='_compute_refund_related_fields')
+    refunded_order_id = fields.Many2one('pos.order', compute='_compute_refund_related_fields')
     has_refundable_lines = fields.Boolean('Has Refundable Lines', compute='_compute_has_refundable_lines')
-    refunded_orders_count = fields.Integer(compute='_compute_refund_related_fields')
     ticket_code = fields.Char(help='5 digits alphanumeric code to be used by portal user to request an invoice')
 
     @api.depends('lines.refund_orderline_ids', 'lines.refunded_orderline_id')
     def _compute_refund_related_fields(self):
         for order in self:
-            order.refund_orders_count = len(order.mapped('lines.refund_orderline_ids.order_id'))
-            order.is_refunded = order.refund_orders_count > 0
-            order.refunded_order_ids = order.mapped('lines.refunded_orderline_id.order_id')
-            order.refunded_orders_count = len(order.refunded_order_ids)
+            order.refunded_order_id = order.mapped('lines.refunded_orderline_id.order_id')
+            order.is_refunded = bool(order.refunded_order_id.id)
```

Comment:
I think you mix up what is_refunded is. Before it was a flag telling if the current order has been refunded in other orders and you transform it into a flag telling whether or not the current order is refunding another order. Is it intentional ?

Reply:
Good catch! my bad. I will change it back